packages/frontend/services/product.ts

```diff
@@ -70,3 +75,9 @@ export const deleteFromFavorites = (productId: string) =>
   http.delete({
     url: `${ApiRoutes.PRODUCTS}${ProductApiRoutes.FAVORITE}?productId=${productId}`,
   });
+
+export const placeBidRequest = (data: CreateBidRequest) =>
+  http.post({
+    url: `${ApiRoutes.BIDS}`,
+    body: { ...data },
```

Comment:
you don't need this spread

Reply:
i do
https://github.com/BinaryStudioAcademy/bsa-2022-vse-bude/runs/8111865047?check_suite_focus=true